src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Figures {
```

Comment:
enums should not be named in plural form
```suggestion
public enum FigureType {
```